WyimaginowaneKoniki/Kongres
# At given date assign randomly reviewers to scientific works

I configured Quartz package, for executing jobs/function at a given time.
For now, I only implemented a function for assigning reviewers to scientific works .

What you should to test this PR?
1. Change date and time in appsettings.json in EndOfRegister (at what time job should execute)
2. You should add to database unique 4 reviewers and unique 4 authors (author and reviewer only one can have same email for each situation. For example, you can have 4 users who have 2 accounts: reviewer and author). Reviewers and works should have  same category of specialization.
3. Check in db that everything went well ;)
